Content.Server/Nutrition/Components/HungerComponent.cs

```diff
@@ -206,10 +198,10 @@ public void OnUpdate(float frametime)
             {
                 // --> But they are not dead yet.
                 var damage = 2 * frametime;
-                _acumulatedDamage += damage - ((int) damage);
+                _accumulatedDamage += damage - ((int) damage);
                 damageable.TryChangeDamage(_damageType, (int) damage);
-                if (_acumulatedDamage >= 1) {
-                    _acumulatedDamage -= 1;
+                if (_accumulatedDamage >= 1) {
+                    _accumulatedDamage -= 1;
                     damageable.TryChangeDamage(_damageType, 1, true);
                 }
```

Comment:
It should be obvious, but we should probably add a note in an issue or a TODO that this needs to be redone when we get fixed decimal point damage.

Reply:
For now I'll just added a TODO DAMAGE UNITS in HealthChange, ThirstComponent, and HungerComponent.
I'll also rename _accumulatedHealth in HealthChange to be consistent.